tools/scripts/upload_to_artifactory.py

```diff
@@ -0,0 +1,147 @@
+# SPDX-License-Identifier: (GPL-1.0-only OR BSD-2-Clause)
```

Comment:
was this script copied from the linux repo? we don't have such Licenses on no-os repository, as far as I know.